libs/datamodel/core/tests/reformat/reformat.rs

```diff
@@ -402,6 +402,72 @@ fn incomplete_field_definitions_in_a_model_must_not_get_removed() {
     assert_reformat(input, input);
 }
 
+#[test]
+fn new_lines_inside_block_above_field_must_stay() {
+    let input = r#"model Post {
```

Comment:
I think the test could be a bit simpler if you remove the extra spaces from the `id` field. Then you could declarer `let expected = input`. This would make those tests quite a bit easier to read imo. The assert would basically literally say that the format should not do anything.
Just a suggestion your call. :-)

Reply:
Ah yes, that'd a lot easier. I'll restore the branch and make these small changes 👍

Reply:
created a PR here: https://github.com/prisma/prisma-engines/pull/968